packages/web-components/fast-foundation/src/radio/radio.ts

```diff
@@ -82,14 +82,14 @@ export class Radio extends FormAssociated<HTMLInputElement> implements RadioCont
      * @public
      */
     @observable
-    public defaultChecked: boolean;
+    public defaultChecked: boolean | undefined = undefined;
```

Comment:
You don't really need an assignment here. Declaring the property but not assigning it implicitly sets the value to `undefined`

Reply:
I was just trying to get rid of the red in vscode